prog1.py

```diff
@@ -1,28 +1,66 @@
+#!/usr/bin/python
+
+import argparse
 import psutil
 import subprocess
 import re
 import time
+import multiprocessing
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-n", "--number", type=int, help="Number of files to create")
+parser.add_argument("-s", "--size", type=int, help="Size of file to create in MB")
+parser.add_argument("-m", dest='min_free', type=int, help="Minimum disk free space in MB")
+parser.add_argument("-l", dest="list",  action='store_true', help="List available mount points. Should be used with \'-m\' option")
+parser.add_argument("-p", dest='mount_point', type=str, help="Mount point to store files")
 
-file_num = 3
-file_size = 4096
-#free_space = 
+args = parser.parse_args()
```

Comment:
This call needs to be moved into `if __name__ == "__main__"` block, which is executed only when this Python module is running as a standalone program.
All statements in the "global" scope are executed immediately when the module is imported by `module_import`. It's not a critical issue, but a suggestion for a future.